### DISCUSSION TITLE
Interruttore TRADFRI in hub smarthings, integrazione su HA

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#1260] [HUB: Home Assistant]
Ciao a tutti, 

ho integrato il remote controller di IKEA (quello tondo, modello E1810) sul mio hub smarthings, che normalmente utilizzo come hub per i dispositivi samsung e zigbee.

Il remote è stato riconosciuto senza problemi sia dall'hub che da Home asisstant attraverso l'integrazione di smarthing (installato in versione CORE via docker).
Però, Home Assistant non vede tutte le entità che mi aspettavo: vede solamente il livello batteria. Questo mi rende impossibile sfruttare il remote per delle automazioni interessanti. Anche se ho installato zigbee2mqtt e dispongo di chiavetta CC2531, vorrei evitare questa via perchè come sappiamo la chiavetta è deboluccia e devo posizionare il remote parecchio distante.

Avete consigli in merito?
Grazie

**Comment by KC-inDomus** [comment]
prova il telecomando con zigbee2mqtt e appura che davvero la distanza rappresenti un problema. diversamente, prendi un repeater di segnale (ti basta una lampadina zigbee)

**Reply by calcioscacchi** [reply]
la tua risposta è un po' una botta al cuore ma farò un tentativo.

UPDATE: 
il remote è stato messo vicino alla chiavetta, per ora, e viene riconosciuto. 
Purtroppo però anche in questo caso le entità che rileva sono legate alla batteria.
